src/controls/Link/index.js

```diff
@@ -56,15 +56,15 @@ class Link extends Component {
     this.signalExpanded = !this.state.expanded;
```

Comment:
Currently this component is not passing the complete currentState to the custom Link component

link object does not contain attributes property

https://github.com/geosolutions-it/react-draft-wysiwyg/blob/402b93f76cf126ecaff5d828691d54c56e022ee2/src/controls/Link/index.js#L204


so probably this need to be updated
https://github.com/geosolutions-it/react-draft-wysiwyg/blob/402b93f76cf126ecaff5d828691d54c56e022ee2/src/controls/Link/index.js#L73-L94